### PR TITLE
Release v10.0.0

### PR DESCRIPTION
**Pull request by duboisp**
GCWeb Release v10.0.0 (compiled with wet-boew v4.0.44)

TODO:
* [x] Update the reference to wet-boew v4.0.44 in package-lock.json
* [x] Merge PR #1899
* [x] Merge PR #1897

### CODE REVIEW

File: index-en.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ lang: en
 		<p>The page templates and design patterns below comprise a reference implementation of the <a href="https://design.canada.ca">Canada.ca design system</a>, including the mandatory requirement of the Content and Information Architecture (C&amp;IA) Specification. Government of Canada departments and agencies can contribute additional patterns and templates via <a href="https://github.com/wet-boew/GCWeb">GCWeb github repository</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.5.0</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v9.5.0">Release notes</a>)</small></p>
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.0.0</strong></a><br />
```
Comment (GormFrank) [inline index-en.md:15]:
```suggestion
		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.0.0 (STR)</strong></a><br />
```

File: index-fr.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ lang: fr
 		<p>Les gabarits et les conceptions communes si dessous sont une référence d'implémentation du <a href="https://conception.canada.ca">Système de conception de Canada.ca</a>, incluant les exigences obligatoire de la spécifications du contenu et de l’architecture de l'information (C&amp;AI) pour Canada.ca. Les ministères et organisme du gouvernement du Canada peuvent y contribuer en publiant leur modèle et leur conception commune via le <a href="https://github.com/wet-boew/GCWeb">dépôt github de GCWeb</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v9.5.0</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v9.5.0">Note de version</a>)</small></p>
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v10.0.0</strong></a><br />
```
Comment (GormFrank) [inline index-fr.md:15]:
```suggestion
		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v10.0.0 (DCT)</strong></a><br />
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
Small changes

File: index-en.md (hunk 1 above)
Comment (duboisp) [reply index-en.md:15]: We didn't added STR in the index HTML page, but only in the release note

**Review by GormFrank — APPROVED** [review]
Approved as long as the To Do list is complete.

**Pull request merged**